bolt/lib/nonlinear_solver/tests/test_interp_2d.py

```diff
@@ -58,5 +58,4 @@ def test_f_interp_2d():
         ) / f_analytic[3:-3, 3:-3].elements()
```

Comment:
Do not hardcode the number of ghost zones here `f_analytic[3:-3, 3:-3]`. Use `f_analytic[N_ghost:-N_ghost, N_ghost:-N_ghost]` instead.